src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -318,89 +326,70 @@ public List<JobModel> enumerateJobs(
         throw new IllegalArgumentException(String.format("Unrecognized direction %s", direction));
     }
 
-    // Filter on FQ class name if specified
-    if (!StringUtils.isEmpty(className)) {
-      filter.addFilterFlightClass(FlightFilterOp.EQUAL, className);
-    }
-
-    List<FlightState> flightStateList;
-    boolean canListAnyJob = checkUserCanListAnyJob(userReq);
+    List<FlightState> flightStateList = null;
     try {
-      if (canListAnyJob) {
-        flightStateList = stairway.getFlights(offset, limit, filter);
-      } else {
-        // Filter out sub-flights
-        filter.addFilterFlightClass(FlightFilterOp.NOT_EQUAL, FileIngestWorkerFlight.class);
-
-        flightStateList = getAccessibleFlights(offset, limit, filter, userReq);
-      }
-    } catch (InterruptedException ex) {
-      throw new JobServiceShutdownException("Job service interrupted", ex);
+      flightStateList = stairway.getFlights(offset, limit, filter);
+    } catch (InterruptedException e) {
+      throw new JobServiceShutdownException("Job service interrupted", e);
     }
     return flightStateList.stream()
         .map(this::mapFlightStateToJobModel)
         .collect(Collectors.toList());
   }
 
-  private List<FlightState> getAccessibleFlights(
-      int offset, int limit, FlightFilter filter, AuthenticatedUserRequest userReq)
-      throws InterruptedException {
-
-    String nextPageToken = null;
-    HashMap<String, List<String>> roleMap = new HashMap<>();
-    ArrayList<FlightState> flightStateList = new ArrayList<>();
-    int numTimesQueried = 0;
-    while (flightStateList.size() < offset + limit && numTimesQueried < MAX_FLIGHT_SEARCH_QUERIES) {
```

Comment:
The constants `MAX_FLIGHT_SEARCH_QUERIES` `FLIGHT_SEARCH_BATCH_SIZE` are no longer used and can be removed.

Reply:
ah great call!